Backend/prisma/schema.prisma

```diff
@@ -0,0 +1,65 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+
+model User {
+  id                    String            @id @default(uuid())
+  firstName             String
+  lastName              String
+  username              String            @unique
+  password              String
+  email                 String            @unique
+  spotifyAccount        SpotifyAccount?
+  nowListening          NowListening[]
+  initiatedFriendships  Friendship[]      @relation("FriendshipInitiator")
+  receivedFriendships   Friendship[]      @relation("FriendshipReceiver")
```

Comment:
are these friend requests received, but not yet accepted?

Reply:
yes